### PR TITLE
✨ Admin setting UI revemp

### PR DESCRIPTION
**Pull request by williamchong**
image
image

image

### CODE REVIEW

File: likecoin/js/admin-settings/src/App.js [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
+  const {
+    DBUserCanEditOption,
+  } = useSelect((select) => select(SITE_LIKER_INFO_STORE_NAME).selectSiteLikerInfo());
   return (
-    <div className="App">
-      <Switch>
-        <Route path="/" exact>
-          <MainSettingPage />
+    <div className="wrap">
+      <h2> </h2>
```
Comment (nwingt) [inline likecoin/js/admin-settings/src/App.js:20]:
```suggestion
```

File: likecoin/js/admin-settings/src/components/AdvancedWidgetSetting.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        'Allow editors to customize display setting per post',
+        'likecoin',
+      )}
+      checkRef={perPostOptionEnabledRef} />
+    <a
+      href='#'
+      onClick={handleResetDefault}
+    >
+      {__('Reset to default', 'likecoin')}
+    </a>
```
Comment (nwingt) [inline likecoin/js/admin-settings/src/components/AdvancedWidgetSetting.js:53]: Should indent children of fragment element

File: likecoin/js/admin-settings/src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <div style={{ margin: '5px', padding: '20px' }}>
+      <a style={{
+        padding: '5px',
+        border: 'solid 1px',
+        borderRadius: '5px',
+      }} href="https://liker.land/dashboard" rel="noopener noreferrer" target="_blank">
+        {__('Your Portfolio', 'likecoin')}
+      </a>
+    </div>
+  </header>;
```
Comment (nwingt) [inline likecoin/js/admin-settings/src/components/Header.js:17]:
```suggestion
  return (
    <header style={{ display: 'flex' }}>
      <img src={Logo} alt={__('liker.land logo', 'likecoin')}/>
      <div style={{ flex: 1 }}></div>
      <div style={{ margin: '5px', padding: '20px' }}>
        <a
          style={{
            padding: '5px',
            border: 'solid 1px',
            borderRadius: '5px',
          }}
          href="https://liker.land/dashboard"
          rel="noopener noreferrer"
          target="_blank"
        >{__('Your Portfolio', 'likecoin')}</a>
      </div>
    </header>
  );
```

File: likecoin/js/admin-settings/src/components/PublishSetting.js [hunk 4]
```diff
@@ -158,61 +137,56 @@ function PublishSettingPage() {
...
+            <CheckBox
+              checked={siteMattersAddFooterLink}
+              handleCheck={setSiteMattersAddFooterLink}
+              title={__('Add post link in footer', 'likecoin')}
+              details={__('Add post link in footer', 'likecoin')}
+              checkRef={siteMattersAddFooterLinkRef} />
+          </tbody>
+        </table></>
+    )}
+  </>
```
Comment (nwingt) [inline likecoin/js/admin-settings/src/components/PublishSetting.js:188]:
```suggestion
            mattersLoginError={mattersLoginError} />}
          <hr />
          <Section title={__('Matters connection status', 'likecoin')} />
          <MattersStatusTable
            siteMattersId={siteMattersId}
            handleMattersLogout={handleMattersLogout} />
          <Section title={__('Publish to Matters', 'likecoin')} />
          <table className="form-table" role="presentation">
            <tbody>
              <CheckBox
                checked={siteMattersAutoSaveDraft}
                handleCheck={setSiteMattersAutoSaveDraft}
                title={__('Auto save draft to Matters', 'likecoin')}
                details={__('Auto save draft to Matters', 'likecoin')}
                checkRef={siteMattersAutoSaveDraftRef} />
              <CheckBox
                checked={siteMattersAutoPublish}
                handleCheck={setSiteMattersAutoPublish}
                title={__('Auto publish post to Matters', 'likecoin')}
                details={__('Auto publish post to Matters', 'likecoin')}
                checkRef={siteMattersAutoPublishRef} />
              <CheckBox
                checked={siteMattersAddFooterLink}
                handleCheck={setSiteMattersAddFooterLink}
                title={__('Add post link in footer', 'likecoin')}
                details={__('Add post link in footer', 'likecoin')}
                checkRef={siteMattersAddFooterLinkRef} />
            </tbody>
          </table></>
      )}
    </>
```

File: likecoin/js/admin-settings/src/components/PublishSetting.js [hunk 5]
```diff
@@ -158,61 +137,56 @@ function PublishSettingPage() {
...
+      handleCheck={setShowMatters}
+      title={__('Matters', 'likecoin')}
+      details={__('Publish to Matters.news', 'likecoin')} />
+    {showMatters && (
+      <><Section title={__('Login with Matters ID', 'likecoin')} />
+        <MattersDescription />
+        {!siteMattersId && <MattersLoginTable
           loginHandler={loginHandler}
           mattersIdRef={mattersIdRef}
           mattersPasswordRef={mattersPasswordRef}
```
Comment (nwingt) [inline likecoin/js/admin-settings/src/components/PublishSetting.js:157]:
```suggestion
  return (
    <>
      <Section title={__('Publish to other platforms', 'likecoin')} />
      <p>{__('LikeCoin plugin can help you to publish you post to other platform.', 'likecoin')}</p>
      <CheckBox
        checked={showMatters}
        handleCheck={setShowMatters}
        title={__('Matters', 'likecoin')}
        details={__('Publish to Matters.news', 'likecoin')}
      />
      {showMatters && (
        <>
          <Section title={__('Login with Matters ID', 'likecoin')} />
          <MattersDescription />
          {!siteMattersId && <MattersLoginTable
            loginHandler={loginHandler}
            mattersIdRef={mattersIdRef}
            mattersPasswordRef={mattersPasswordRef}
```

File: likecoin/js/admin-settings/src/components/WebMonetization/WebMonetizationSetting.js [hunk 6]
```diff
@@ -1,58 +1,42 @@
...
         title={__('Web Monetization', 'likecoin')}
-        details={__('Show Web Monetization settings', 'likecoin')} />
+        details={__('Enable', 'likecoin')} />
       {showWebMonetization && (
-        <><WebMonetizationDescription /><form onSubmit={confirmHandler}>
-          <table className="form-table" role="presentation">
+        <><table className="form-table" role="presentation">
             <tbody>
               <tr>
                 <th scope="row">
```
Comment (nwingt) [inline likecoin/js/admin-settings/src/components/WebMonetization/WebMonetizationSetting.js:42]: Please correct the indentation 

```jsx
<>
   {children}
</>
```

File: likecoin/js/admin-settings/src/pages/LikerIdSettingLayout.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  const {
+    DBUserCanEditOption,
+  } = useSelect((select) => select(SITE_LIKER_INFO_STORE_NAME).selectSiteLikerInfo());
+  return <><nav style={navStyle}>
+      {DBUserCanEditOption && <NavLink style={tableStyleFn} to="" end={true}>{__('Website Liker ID', 'likecoin')}</NavLink>}
+      <NavLink style={tableStyleFn} to="user">{__('Your Liker ID', 'likecoin')}</NavLink>
+    </nav>
+    <Outlet />
+  </>;
+}
```
Comment (nwingt) [inline likecoin/js/admin-settings/src/pages/LikerIdSettingLayout.js:34]:
```suggestion
  return (
    <>
      <nav style={navStyle}>
        {DBUserCanEditOption && <NavLink style={tableStyleFn} to="" end={true}>{__('Website Liker ID', 'likecoin')}</NavLink>}
        <NavLink style={tableStyleFn} to="user">{__('Your Liker ID', 'likecoin')}</NavLink>
      </nav>
      <Outlet />
    </>
  );
}
```

File: likecoin/js/admin-settings/src/pages/MainSettingLayout.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+  const {
+    DBUserCanEditOption,
+  } = useSelect((select) => select(SITE_LIKER_INFO_STORE_NAME).selectSiteLikerInfo());
+  return <><nav style={navStyle}>
+      {DBUserCanEditOption && (<NavLink style={tableStyleFn} to="" end={true}>{__('General', 'likecoin')}</NavLink>)}
+      {DBUserCanEditOption && (<NavLink style={tableStyleFn} to="advanced">{__('Advanced', 'likecoin')}</NavLink>)}
+      <NavLink style={tableStyleFn} to="about">{__('About', 'likecoin')}</NavLink>
+    </nav>
+    <Outlet />
+  </>;
```
Comment (nwingt) [inline likecoin/js/admin-settings/src/pages/MainSettingLayout.js:36]:
```suggestion
  return (
    <>
      <nav style={navStyle}>
        {DBUserCanEditOption && (<NavLink style={tableStyleFn} to="" end={true}>{__('General', 'likecoin')}</NavLink>)}
        {DBUserCanEditOption && (<NavLink style={tableStyleFn} to="advanced">{__('Advanced', 'likecoin')}</NavLink>)}
        <NavLink style={tableStyleFn} to="about">{__('About', 'likecoin')}</NavLink>
      </nav>
      <Outlet />
    </>
  );
```

File: likecoin/js/admin-settings/src/pages/UserLikerIdSettingPage.js [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+      {savedSuccessful && (
+        <SettingNotice
+          text={__('Settings Saved', 'likecoin')}
+          className="notice-success"
+          handleNoticeDismiss={handleNoticeDismiss}
+        />
+      )}
+      <form onSubmit={updateLikerIdHandler}>
+        <Section title={__('Your Liker ID', 'likecoin')} />
+            <p>{__('This is your Liker ID, which is used to display LikeCoin button when post is not registered on ISCN yet.', 'likecoin')}</p>
```
Comment (nwingt) [inline likecoin/js/admin-settings/src/pages/UserLikerIdSettingPage.js:60]:
```suggestion
        <Section title={__('Your Liker ID', 'likecoin')} />
        <p>{__('This is your Liker ID, which is used to display LikeCoin button when post is not registered on ISCN yet.', 'likecoin')}</p>
```

**Review by nwingt — CHANGES_REQUESTED** [review]

File: likecoin/js/admin-settings/src/App.js (hunk 1 above)
Comment (williamchong) [reply likecoin/js/admin-settings/src/App.js:20]: This is `h2` under `.wrap` is needed for displaying notice correctly, will add comment

**Pull request merged**